tests/test_common.py

```diff
@@ -744,6 +744,24 @@ def test_find_closest_point():
     assert len(index) == 5
 
 
+def test_find_closest_cell():
+    mesh = pyvista.Wavelet()
+    node = np.array([0, 0.2, 0.2])
+
+    with pytest.raises(TypeError):
+        mesh.find_closest_cell([1, 2])
+
+    with pytest.raises(TypeError):
+        # allow Sequence but not Iterable
+        mesh.find_closest_cell({1, 2, 3})
+
+    index = mesh.find_closest_cell(node)
+    assert isinstance(index, int)
+    # Make sure we can fetch that point
+    closest = mesh.points[index]
+    assert len(closest) == 3
```

Comment:
We're getting a cell instead of a point index.  I just removed this.  If we really wanted to test this, we'd get the center of a cell and then make sure we matched that index.